client/src/pages/private/SiteTable.js

```diff
@@ -110,48 +142,114 @@ export default ({ sites }) => {
   },[healthAuthorities, fetchedRows, sites]);
 
+  const defaultOnClose = () => {
+    setActiveModalForm(null);
+  };
+
   return (
-    <Grid container alignContent="flex-start" justify="flex-start" alignItems="center" direction="column">
-      <Grid container alignContent="flex-start" justify="flex-start" alignItems="center" direction="row">
-        <Grid item>
-          <Box pl={2} pr={2} pt={1}>
-            <Typography variant="body1" gutterBottom>
-              Filter:
-            </Typography>
-          </Box>
-        </Grid>
-        <Grid item>
-          <Box minWidth={180}>
-            <TableFilter
-              onFilter={(filteredRows) => setRows(filteredRows)}
-              values={healthAuthorities}
-              rows={fetchedRows}
-              label="Health Authority"
-              filterField="healthAuthority"
-            />
-          </Box>
+    <>
+    <Dialog
+        title={`Create Site`}
+        open={activeModalForm != null}
+        onClose={defaultOnClose}
+      >
+        {activeModalForm === 'new-site' && <NewSiteForm
+          initialValues={{
+            siteId: '',
+            siteName: '',
+            registeredBusinessName: '',
+            address: '',
+            city: '',
+            postalCode: '',
+            healthAuthority: '',
+            allocation: '',
+            operatorName: '',
+            operatorContactFirstName: '',
+            operatorContactLastName: '',
+            operatorPhone: '',
+            operatorEmail: '',
+            siteContactFirstName: '',
+            siteContactLastName: '',
+            siteContactPhone: '',
+            siteContactEmail: '',
+          }}
+          validationSchema={CreateSiteSchema}
+          onSubmit={(values) => {
+            handleSiteCreate({
+              siteId: parseInt(values.siteId),
+              siteName: values.siteName,
+              registeredBusinessName: values.registeredBusinessName,
+              address: values.address,
+              city: values.city,
+              postalCode: values.postalCode,
+              healthAuthority: values.healthAuthority,
+              allocation: parseInt(values.allocation),
+              operatorName: values.operatorName,
+              operatorContactFirstName: values.operatorContactFirstName,
+              operatorContactLastName: values.operatorContactLastName,
+              operatorPhone: values.operatorPhone,
+              operatorEmail: values.operatorEmail,
+              siteContactFirstName: values.siteContactFirstName,
+              siteContactLastName: values.siteContactLastName,
+              siteContactPhone: values.siteContactPhone,
+              siteContactEmail: values.siteContactEmail,
+            });
+          }}
+          onClose={defaultOnClose}
+        />}
+      </Dialog>
+      <Grid container alignContent="flex-start" justify="flex-start" alignItems="center" direction="column">
```

Comment:
below here in this file is just new indentation due to being wrapped in the parent fragment now - except for the new button added starting line 221